Open Source Development/README.MD

```diff
@@ -31,6 +31,10 @@ Open source development involves collaborative creation of software where the so
     <td><a href="https://opensource.org/">Open Source Initiative(OSI)</a></td>
     <td>The OSI website provides information on the principles of open source software and licenses, helping developers understand the legal and ethical aspects of open source development.</td>
   </tr>
+  <tr>
+    <td><a href="https://bitbucket.org/">Bitbucket</a></td>
+    <td>Bitbucket is a cloud-based Git-based service that allows developers to store, manage, and track code changes. It's designed for professional teams, providing a central place to collaborate on source code, plan projects, and test and deploy code.</td>
+  </tr>
```

Comment:
Moved it instead in the [Open Source Technologies](https://github.com/jfmartinz/ResourceHub/tree/main/Open%20Source%20Development#open-source-technologies). Also Github Move it to that table. 
![image](https://github.com/jfmartinz/ResourceHub/assets/129386460/13fc73c3-6e26-4ae5-8135-7be435361848)